lib/Subscribers/InactiveSubscribersController.php

```diff
@@ -71,7 +73,8 @@ private function deactivateSubscribers(Carbon $threshold_date, $batch_size, $sta
 
     // We take into account only emails which have at least one opening tracked
     // to ensure that tracking was enabled for the particular email
-    $inactives_task_ids_table = sprintf("
+    if (!$this->inactives_task_ids_table_created) {
```

Comment:
This makes the query log cleaner when debugging. The behavior itself is MySQL version-dependent: before 5.5.6 the `SELECT` part was still executed and rows were inserted each time, since 5.5.6 the select is not performed (https://dev.mysql.com/doc/refman/5.5/en/create-table-select.html). If I'm not mistaken, TDN runs an older version which executes each time.

Reply:
Wow. I that is really good to know. According to Mixpanel 5.5 is the top version.